content/blog/2020-05-19-learnr/index.markdown

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Teach R with learnr: a powerful tool for remote teaching'
+author:
+  - '[Allison Horst](https://twitter.com/allison_horst)'
+date: '2020-05-19'
+categories:
+  - teach
+tags:
+  - rmarkdown
+  - learnr
+description: |
+  'Why and how I created a learnr tutorial on *Exploring missing data in naniar*'
+slug: learnr-for-remote
+photo:
+  url: https://github.com/allisonhorst/stats-illustrations
+  author: Allison Horst
+---
+
+
+
+The unexpected switch to remote instruction has been a challenge for teachers and learners, and has inspired - or for some, forced - the use of innovative teaching methods and tools. Coding lessons pose a unique challenge for remote instruction as teachers are unavailable for in-person troubleshooting, and beginning students are unlikely to have developed debugging or reprex-building skills. On top of coding-related challenges, students might not have access to a device on which they can install R and RStudio.
+
+So, how can we create live coding lessons to include participants who... 
+
+- ...have not used or installed R and RStudio?
+- ...are unable to quickly get in-person troubleshooting help during live lessons? 
+
+The [learnr](https://rstudio.github.io/learnr/) package offers a solution to both. In this post, I describe why and how I created a learnr tutorial on [*Exploring missing data in naniar*](https://allisonhorst.shinyapps.io/missingexplorer/) with both first-time and experienced R-users in mind.
+
+
+## 1. Approachable for first-time R users
+
+For my remote lesson on exploring missing data (using [Dr. Nick Tierney](https://www.njtierney.com/)’s [`naniar`](https://cran.r-project.org/web/packages/naniar/vignettes/getting-started-w-naniar.html) package), I expected a combination of: 
+
+- Advanced R users
+- Beginning R learners
+- Attendees who had never used or seen R & RStudio before
+
+Usually, I have students fork and clone materials from a GitHub repo so that they can follow along in the RStudio IDE (Desktop, or through [RStudio Cloud](https://rstudio.cloud/)). For my live lesson, I wanted *everyone* - even those who had never used or seen R/RStudio before - to be able to write and run R code successfully, and I didn’t want those first-time R-users to need to navigate the IDE at the same time. 
+
+### learnr tutorials look and feel like a simple website 
+
+I love the RStudio IDE. If you’re reading this, you probably do, too. But try to remember what it felt like the first time you ever opened RStudio. There is a lot there, and it takes *a long time* to feel really comfortable navigating it. Eventually, most R-users will need to become good friends with the RStudio IDE. But for first-time users working remotely and without hands-on help, a learnr tutorial - which looks like a simple, easy-to-navigate website - could be a more familiar and welcoming entry. 
```

Comment:
"you opened it" instead of "you ever opened RStudio", and remove the emphasis on "a long time".